ui/component/notification/view.jsx

```diff
@@ -97,6 +107,52 @@ export default function Notification(props: Props) {
       icon = <Icon icon={ICONS.NOTIFICATION} sectionIcon />;
   }
 
+  let notificationLink = formatLbryUrlForWeb(notificationTarget);
+  let urlParams = new URLSearchParams();
+  if (isCommentNotification && notification_parameters.dynamic.hash) {
+    urlParams.append('lc', notification_parameters.dynamic.hash);
+  }
+
+  let channelName = channelUrl && ('@' + channelUrl.split('@')[1].split('#')[0]);
+
+  const notificationTitle = notification_parameters.device.title;
+  const titleSplit = notificationTitle.split(' ');
+  let fullTitle = [' '];
+  let uriIndicator;
+  const title = titleSplit.map((message, index) => {
+    if (channelName === message) {
+      uriIndicator = (
+      <UriIndicator uri={channelUrl} link />
+      );
+      fullTitle.push(' ');
+      const resultTitle = fullTitle;
+      fullTitle = [' '];
+
+      return [resultTitle.join(' '), uriIndicator];
+    } else {
+      fullTitle.push(message);
+
+      if (index === (titleSplit.length - 1)) {
+        return (
+          <LbcMessage>{fullTitle.join(' ')}</LbcMessage>
+        );
+      }
+    }
+  });
```

Comment:
Feels like can be improved, but will pass for now so as not to stall the PR further.

With a little more work, I think we can avoid the unnecessary loop and also enable localization at the same time using something like the following (incomplete example).  It'll also prevent the need to put a blanket `LbcMessage` for notifications that don't need it. We can also customize the phrase like adding quotes to the title for readability.

```
function generateTitle(rule, params) {
    switch (rule) {
      case RULE.COMMENT:
        return (
          <I18nMessage
            tokens={{
              commenter: <UriIndicator uri={notification_parameters.dynamic.comment_author} link />,
              title: notification_parameters.dynamic.claim_title,
            }}
          >
            %commenter% commented on "%title%"
          </I18nMessage>
        );

      case RULE.CREATOR_SUBSCRIBER:
        // TODO
        break;
      case RULE.CREATOR_COMMENT:
        // TODO
        break;
      case RULE.COMMENT_REPLY:
        // TODO
        break;
      case RULE.NEW_CONTENT:
      case RULE.NEW_LIVESTREAM:
      case RULE.DAILY_WATCH_AVAILABLE:
      case RULE.DAILY_WATCH_REMIND:
      case RULE.MISSED_OUT:
      case RULE.REWARDS_APPROVAL_PROMPT:
      case RULE.FIAT_TIP:
      default:
        // Use Commentron default
        return params.device.title;
    }
  }

  const title = generateTitle(notification_rule, notification_parameters);

```